examples/guided_onboarding_example/public/components/main.tsx

```diff
@@ -266,12 +272,7 @@ export const Main = (props: MainProps) => {
             <EuiFormRow label="Guide" helpText="Select a guide">
               <EuiSelect
                 id="guideSelect"
-                options={[
-                  { value: 'observability', text: 'observability' },
-                  { value: 'security', text: 'security' },
-                  { value: 'search', text: 'search' },
-                  { value: 'testGuide', text: 'test guide' },
-                ]}
+                options={selectOptions}
```

Comment:
This is outside the scope of this PR, but we might want to make this a free-form field once more guides are added.